src/main/java/nl/avans/drivio/model/HydrogenCar.java

```diff
@@ -1,16 +1,72 @@
 package nl.avans.drivio.model;
 
+import org.springframework.stereotype.Component;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table
 public class HydrogenCar extends Car {
-    int range;
-    double kgPer100Km;
-    int tankSpeed;
-    int tankSize;
+    @Column(name = "RangeCar")
+    private int range;

```

Comment:
Zullen we dezelfde naam als in de database aanhouden? Dus in de klasse ook RangeCar of CarRange ervan maken?

Mening?

Reply:
Ik heb de wijziging uitgevoerd (zie commit [[Feat: Changed the requested changes from PR#25](https://github.com/leonvsc/Drivio/pull/25/commits/8fa43389fecb9ea25438afd652b9d161e75421ff))](https://github.com/leonvsc/Drivio/pull/25/commits/8fa43389fecb9ea25438afd652b9d161e75421ff)